packages/css-es-modules/src/collect-styles.ts

```diff
@@ -3,16 +3,14 @@ import { CSS_GLOBAL_KEY, CSS_LOCALS_KEY } from './inject-styles';
 
 // globals declaration to avoid the typescript compile errors
 declare global {
+    // Using `string | true` as second record generic because Window is type of `Window & typeof globalThis;`
+    var $CSS$IN$JS$GLOBALS$: Record<string, string | true>;
```

Comment:
Not sure how to declare `var [CONST_VALUE]: Record<string, string | true>;` that's why I used direct values